checkpointing/tensorflow/checkpoint.sub

```diff
@@ -0,0 +1,30 @@
+
+universe = vanilla
+log = checkpoint.log
+
+executable = run.sh
+
+checkpoint_exit_code = 85 
+transfer_checkpoint_files = checkpoint.h5
```

Comment:
add a comment about what this does? something like: 
`# indicate which checkpointing files should be preserved to facilitate restarts in case of interruption`